docs/pages/enroll-resources/database-access/enroll-aws-databases/rds-oracle.mdx

```diff
@@ -0,0 +1,378 @@
+    ```
+  </TabItem>
+  <TabItem label="RHEL 9">
+    ```code
+    $ sudo curl -o /etc/yum.repos.d/msprod.repo https://packages.microsoft.com/config/rhel/9/prod.repo
+    $ sudo ACCEPT_EULA=Y yum install -y adutil
+    ```
+  </TabItem>
+  <TabItem label="Other">
+  If `adutil` is not available for your distribution, you can use `ktutil` instead. It should be available as a standard Kerberos utility.
+  ```code
+  $ ktutil
+ktutil: addent -password -p alice@EXAMPLE.COM -k 1 -e aes256-cts-hmac-sha1-96
+Password for alice@EXAMPLE.COM: [enter password here]
+ktutil: wkt teleport.keytab
+ktutil: quit
+  ```
+  </TabItem>
+</Tabs>
+
+Log in to Active Directory using the `kinit` command:
+
+```code
+$ kinit admin@EXAMPLE.COM
+```
+
+Use the `adutil keytab create` command to generate keytab entries for each
+Active Directory user that will be connecting to the SQL Server database:
+
+```code
+$ adutil keytab create teleport.keytab alice
+$ adutil keytab create teleport.keytab bob
+```
+
+You will be prompted to enter each user's password. All keytab entries will
+be merged into the same `teleport.keytab` file.
+
+<Admonition type="note" title="Assign Service Principal Names">
+  For the `adutil keytab create` command to work, each user account must be
+  assigned a Service Principal Name, otherwise the command will not be able
+  to determine its `kvno` (key version number).
+
+  To check if the user has any SPNs assigned, run the following command on the
+  Windows machine joined to your Active Directory domain:
+
+  ```code
+  $ setspn -L alice
+  ```
+
+  To assign an SPN to a user account, use the following command:
+
+  ```code
+  $ setspn -s user/alice alice
+  ```
+</Admonition>
+
+You can verify entries in the keytab file using `klist` command:
+
+```code
+$ klist -ke teleport.keytab
+Keytab name: FILE:teleport.keytab
+KVNO Principal
+---- --------------------------------------------------------------------------
+   5 alice@EXAMPLE.COM (aes256-cts-hmac-sha1-96)
+   2 bob@EXAMPLE.COM (aes256-cts-hmac-sha1-96)
+```
+
+<Admonition type="warning">
+  You must update the keytab file after updating a user's password to avoid
+  authentication failures.
+</Admonition>
+
+## Step 5/9. Set up the Teleport Database Service
+
+(!docs/pages/includes/tctl-token.mdx serviceName="Database" tokenType="db" tokenFile="/tmp/token"!)
+
+(!docs/pages/includes/install-linux.mdx!)
+
+<Admonition type="note">
+  Teleport Database Service must run on a Linux server joined to the same
+  Active Directory domain as the Oracle instance.
+</Admonition>
+
+Configure the Teleport Database Service. Make sure to update `--proxy` to
+point to your Teleport Proxy Service address and `--uri` to the Oracle
+endpoint.
+
+```code
+$ sudo teleport db configure create \
+    -o file \
+    --token=/tmp/token \
+    --proxy=teleport.example.com:443 \
+    --name=oracle \
+    --protocol=oracle \
+    --uri=oracle-instance.aabbccddeegg.eu-central-1.rds.amazonaws.com:2484 \
```

Comment:
I think we should have a note somewhere to use the SSL port from the option group instead of the default port

Reply:
Good point, I'll add a note.